### ISSUE TITLE
alpine-zigbee2mqtt-v5: timezone: unbound variable

### ISSUE THREAD

**Issue by ohmgh** [tteck/Proxmox#1194]
Using Advanced Settings
Using Container Type: 0
Using Root Password: alpine
Container ID: 103
Using Hostname: alpine-zigbee2mqtt
Using Disk Size: 0.3
Allocated Cores: 1
Allocated RAM: 256
Using Bridge: vmbr0
Using IP Address: 10.0.2.103
Using Gateway IP Address: 192.168.1.2
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Alpine-Zigbee2MQTT LXC using the above advanced settings
bash: line 415: timezone: unbound variable

**Comment by tteck** [comment]
Should have been fixed in 0647dc6

**Comment by tteck** [comment]
No issues here, clear browser cache.

```
 _____   _       __             ___   __  _______  ____________
/__  /  (_)___ _/ /_  ___v5___ |__ \ /  |/  / __ \/_  __/_  __/
  / /  / / __  / __ \/ _ \/ _ \__/ // /|_/ / / / / / /   / /   
 / /__/ / /_/ / /_/ /  __/  __/ __// /  / / /_/ / / /   / /    
/____/_/\__, /_.___/\___/\___/____/_/  /_/\___\_\/_/   /_/     
       /____/ Alpine
 
Using Advanced Settings
Using Container Type: 0
Using Root Password: 03-19-2023
Container ID: 112
Using Hostname: alpine-zigbee2mqtt
Using Disk Size: 0.3
Allocated Cores: 1
Allocated RAM: 256
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Alpine-Zigbee2MQTT LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.253
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.114.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Alpine-Zigbee2MQTT
 ✓ Completed Successfully!
```
